snowplow-demo-app/src/main/java/com/snowplowanalytics/snowplowtrackerdemo/Demo.java

```diff
@@ -298,7 +298,8 @@ private boolean setupWithLocalConfig() {
                 .screenContext(true)
                 .exceptionAutotracking(true)
                 .installAutotracking(true)
-                .diagnosticAutotracking(true);
+                .diagnosticAutotracking(true)
+                .userAnonymisation(true);
```

Comment:
This ended up in the "Prepare for release" commit – probably not intentional, right?